bundles/io.openliberty.tools.eclipse.lsp4e/.gitignore

```diff
@@ -1,3 +1,5 @@
-server/*
 lib/*
+server/liberty-langserver/*.jar
```

Comment:
did you mean to include jakarta?  or maybe use a pattern excluding all at once?

Reply:
i am trying to exclude the ignore of the jakarta lang server jar since we are manually shipping it for now until they get the jakarta ls (different from the jakarta jdt) into the maven repo (where we can then get it via a pom dependency and copy it thus no longer needing to manually ship it)